### PR TITLE
Skyrat Syndicate Ports

### PR DESCRIPTION
**Pull request by Moth-lantern**
## About The Pull Request
Ports 2 new outfits including a skirt variant, a new labcoat, and a new pair of gloves from the syndicate folder on skyrat!
These include the sinister yet cute syndicate maid outfit, and two new tactical overalls, along with a menacing black labcoat all from Interdyne and Cybersun's joint fashion line Serpent's kiss.

Also allows these items to be ordered in a new contraband sinister fashion crate, obtained by emagging the cargo console!
syndie maid
Syndie labcoat
syndie overall
syndie overskirt

## Why It's Good For The Game
More unique syndicate items for fashion reason's while being an antag, also cmon somebodies gonna do a killer bit as a syndicate maid.

## Changelog

:cl:
add: New items are now being sold on black market channels, be on the look out for Sinister fashion appearing in the cargo bay.
\:cl:

### CODE REVIEW

File: orbstation/icons/obj/items/clothing/hands.dmi
Comment (flowercuco) [inline orbstation/icons/obj/items/clothing/hands.dmi:1]: whats with this being removed?

File: orbstation/code/objects/items/clothing/skyrat_syndicate.dm [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+/obj/item/clothing/under/syndicateoverskirt
+	name = "tactical overallskirt"
+	desc = "A non-descript and slightly suspicious pair of tactical overalls, it comes in an aesthetically pleasing bloodred"
+	icon_state = "syndicate_overallskirt"
+	has_sensor = NO_SENSORS
+	armor_type = /datum/armor/under_syndicate
+	alt_covers_chest = TRUE
+	icon = 'orbstation/icons/obj/items/clothing/orb_syndicate.dmi'
+	worn_icon = 'orbstation/icons/obj/items/clothing/orb_syndicate_worn.dmi'
```
Comment (flowercuco) [inline orbstation/code/objects/items/clothing/skyrat_syndicate.dm:19]: missing a period

File: orbstation/code/objects/items/clothing/skyrat_syndicate.dm [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+	inhand_icon_state = null
+	blood_overlay_type = "coat"
+	body_parts_covered = CHEST|ARMS
+	armor_type = /datum/armor/toggle_labcoat
+
+//The Cargo Order for said items above
+
+/datum/supply_pack/imports/evilfashion
+	name = "Sinister Fashion Crate"
+	desc = "A box of the latest in imported fashion, featuring items from Cybersun and Interdyne's joint Serpent's Kiss fashion line."
```
Comment (flowercuco) [inline orbstation/code/objects/items/clothing/skyrat_syndicate.dm:64]: i feel like this should have a different name, either Serpent's Kiss Fashion Crate or like "Rival Corp Fashion Crate"? i dont know like, if it is inherently sinister! but thats just me

**Review by flowercuco — CHANGES_REQUESTED** [review]
do all of the .dmi items need to be in their own file? if yes or you just prefer it that way, thats fine. if i dont get back to you plz ping me on discord also

File: orbstation/icons/obj/items/clothing/hands.dmi
Comment (Moth-lantern) [reply orbstation/icons/obj/items/clothing/hands.dmi:1]: AHHH OOPS

**Review by flowercuco — APPROVED** [review]

**Pull request merged**